### PR TITLE
Allow a person to tag buckets.

### PR DESCRIPTION
**Pull request by csc-felipe**
### Description
Gives a person the possibility to add or remove tags from a bucket. Includes the ability to show or hide tags when browsing the buckets.

Please, check the Finnish language :) 

### Related issues
Related to #85.

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### Changes Made
Added backend for updating metadata for a bucket. Added UI for displaying tags for containers, as well as hiding them. There is a new button, called 'Edit', which makes it possible to edit a bucket's tags. When creating a new Bucket, it's possible to add tags to it.

Tags are not displayed in the bucket view (after double click on a bucket). There's no edit button on that page either.

### Testing

- [x] Unit Tests
- [x] Integration Tests

### Mentions
Thanks to @sampsapenna for solving my javascript sync/async mess.

### CODE REVIEW

**Comment by csc-felipe** [comment]
cypress is clicking the wrong switch 
Browse buckets and test operations -- should be able to filter table, adjust display buckets per page and pagination (failed)
.

**Comment by blankdots** [comment]
> cypress is clicking the wrong switch

then we need a better selector for that switch, or something is off with clicking the parent `cy.contains('Paginated').parent().click()` because you put them in the same `control` div, moving them to separate divs should help or finding out a new selector

File: swift_browser_ui_frontend/src/views/Containers.vue [hunk 1]
```diff
@@ -33,12 +33,17 @@
         </option>
       </b-select>
       <div class="control is-flex">
-        <b-switch 
+        <b-switch
```
Comment (blankdots) [inline swift_browser_ui_frontend/src/views/Containers.vue:36]: if you want tests to pass this should help:
```
<div class="control is-flex">
        <b-switch
          v-if="bList.length < 500"
          v-model="isPaginated"
        >
          {{ $t('message.table.paginated') }}
        </b-switch>
      </div>
      <div class="control is-flex">
        <b-switch
          v-model="showTags"
        >
          {{ $t('message.table.showTags') }}
        </b-switch>
      </div>
```

but switches can be grouped also as: https://buefy.org/documentation/switch#styles 

however maybe we need to select that switch a tad different than `cy.contains('Paginated').parent().click()` ... maybe add a `data-testid=` attribute to it

File: swift_browser_ui_frontend/src/common/lang.js [hunk 2]
```diff
@@ -409,18 +416,23 @@ let default_translations = {
       copy: " Kopioi",
       create: "Luo",
       delete: "Poista",
+      edit: "Muokkaa",
+      save: "Tallena",
```
Comment (sampsapenna) [inline swift_browser_ui_frontend/src/common/lang.js:420]: ~Tallena~ -> Tallenna

File: swift_browser_ui_frontend/src/common/lang.js [hunk 3]
```diff
@@ -286,6 +292,7 @@ let default_translations = {
         created: "Luotu",
         folderDetails: "Ei yksityiskohtia kansioille",
         clearChecked: "Poista valinnat",
+        showTags: "Näytä tagit",
```
Comment (sampsapenna) [inline swift_browser_ui_frontend/src/common/lang.js:295]: Probably better as "tägit" with the umlaut – the anglicism is a bit informal/colloquial, but is probably more recognizable for the user

File: swift_browser_ui_frontend/src/common/lang.js [hunk 4]
```diff
@@ -409,18 +416,23 @@ let default_translations = {
...
       delete: "Poista",
+      edit: "Muokkaa",
+      save: "Tallena",
       createContainerButton: "Luo säiliö",
       copysuccess: "Aloitettiin säiliön kopiointi taustalla",
       copyfail: "Säiliön kopiointi epäonnistui",
       renderFolders: "Näytä kansioina",
       container_ops: {
         addContainer: "Luo uusi säiliö",
+        editContainer: "Muokkaaminen säiliö: ",
```
Comment (sampsapenna) [inline swift_browser_ui_frontend/src/common/lang.js:427]: *Muokataan säiliötä:

**Review by sampsapenna — CHANGES_REQUESTED** [review]
Some small alterations to the Finnish upon further inspection, other than that LGTM.

File: swift_browser_ui_frontend/src/common/lang.js (hunk 3 above)
Comment (csc-felipe) [reply swift_browser_ui_frontend/src/common/lang.js:295]: Also on the field label that says Tagit, or only here?

File: swift_browser_ui_frontend/src/views/AddContainer.vue [hunk 5]
```diff
@@ -16,6 +20,24 @@
...
+    >
+      <b-taginput
+        v-model="tags"
+        ellipsis
+        maxlength="20"
+        icon="label"
+        has-counter
+        rounded
+        type="is-primary"
+        :confirm-keys="[',', ';', ':', '.', ' ', 'Tab', 'Enter']"
```
Comment (blankdots) [inline swift_browser_ui_frontend/src/views/AddContainer.vue:39]: can we create a var out of these keys so we can manage them a tad better ?

**Review by blankdots — CHANGES_REQUESTED** [review]
the new POST endpoint "/api/bucket/meta" should be added to the [API.yaml](https://github.com/CSCfi/swift-browser-ui/blob/master/docs/_static/api.yml)

File: swift_browser_ui_frontend/src/common/lang.js (hunk 3 above)
Comment (sampsapenna) [reply swift_browser_ui_frontend/src/common/lang.js:295]: Preferably change that to "Tägit" as well

**Review by blankdots — APPROVED** [review]

**Review by sampsapenna — APPROVED** [review]

**Pull request merged**